### PR TITLE
Background notifications fixed (remade after firebase problem)

### PR DESCRIPTION
**Pull request by LJMNZS**
Closes issue #60 
Background notifications should be working on all browsers
Clicking the notification should bring the website back into focus
The notification should disappear after 5 seconds
Audio notifications not in this PR, still have to decide what sound to use
Hopefully this fixes the Firebase problem with the last PR

### CODE REVIEW

File: source/js/scripts/app.js [hunk 1]
```diff
@@ -176,25 +180,33 @@ function initTimer(timer) {
...
+  let register = null;
+  navigator.serviceWorker.getRegistration()
+    .then((reg) => {
+      register = reg;
+      reg.showNotification('Electric Pomato', pomoAlert)
+        .then(() => register.getNotifications()
+          .then((notifications) => {
+            setTimeout(() => notifications.forEach((notification) => notification.close()), 5000);
+          }));
+    });
```
Comment (markbussard) [inline source/js/scripts/app.js:209]: Maybe try adding a catch here for any errors and console logging it to see whats going on

**Comment by markbussard** [comment]
> 

I'm not sure why those are getting built twice, but if you look in the build folder, it's still just one image for each tomato.png

**Comment by markbussard** [comment]
> @markbussard I think there might be an issue with `npm run build`, the tomato images are getting duplicated in `build` but not `dev` image

I'm not sure why those are getting built twice, but if you look in the build folder, it's still just one image for each tomato.png

**Comment by alanlwang1** [comment]
> > @markbussard I think there might be an issue with `npm run build`, the tomato images are getting duplicated in `build` but not `dev` image
> 
> I'm not sure why those are getting built twice, but if you look in the build folder, it's still just one image for each tomato.png

I'm not sure if it can be a code issue if the same code works for dev

**Comment by alanlwang1** [comment]
> > @markbussard I think there might be an issue with `npm run build`, the tomato images are getting duplicated in `build` but not `dev` image
> 
> I'm not sure why those are getting built twice, but if you look in the build folder, it's still just one image for each tomato.png

In the Network tab, theres still duplicate requests being made despite only being one image

**Comment by markbussard** [comment]
> > > @markbussard I think there might be an issue with `npm run build`, the tomato images are getting duplicated in `build` but not `dev` image
> > 
> > 
> > I'm not sure why those are getting built twice, but if you look in the build folder, it's still just one image for each tomato.png
> 
> In the Network tab, theres still duplicate requests being made despite only being one image image I'm not sure if its a code issue if the dev deployment works Is it possible for the parcel-serviceworker manifest to read the duplicate items from the build?

I'm not sure, and I checked our production page and there is no duplicates issue. I think a possible reason for seeing the duplicates on this branch is that the service worker isn't registering properly

**Comment by markbussard** [comment]
@alanlwang1 Can you try adding a catch to the navigator.serviceWorker.register function in js/scripts/registerSw with a console log to see the registration error?

**Comment by LJMNZS** [comment]
I tried to fix it based on [this link](https://stackoverflow.com/questions/56363401/uncaught-in-promise-typeerror-cannot-read-property-shownotification-of-und). It didn't work so the SW must not be getting registered properly. 
Probably going to try change the registration code ([from here](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration)) next. Hopefully that works

**Review by markbussard — APPROVED** [review]
Looks good to me. Ran through it several times and notifications were showing and focusing screen correctly on click.

**Pull request merged**